### ISSUE TITLE
no spam. even without auto captcha problem persist. some files is possible download, some not.

### ISSUE THREAD

**Issue by hapashtiepa** [setnicka/ulozto-downloader#117]
![Screenshot_2022-09-19_19_02_16](https://user-images.githubusercontent.com/113451474/191073208-f7d8b2e1-0133-4ba2-9179-6d6364e6572d.png)

**Comment by jan-provaznik** [comment]
See https://github.com/setnicka/ulozto-downloader/issues/82

**Comment by zbyna** [comment]
This should be fixed in the up-to-date version. In my opinion, installing it should solve your issue.

**Comment by setnicka** [comment]
@hapashtiepa Did you try a new version of the ulozto-downloader?

**Comment by hapashtiepa** [comment]
No, I will try tomorrow, thanks a milion!
-- 
 Odesláno z Tutanoty, bezpečné e-mailové schránky bez reklam. 



14. 10. 2022 18:18 od ***@***.***:

**Comment by hapashtiepa** [comment]
I actually found time for this. I had version 3.1.0 one month ago as well, without atomated capcha it works. but, I wanted automated. It is very good, but sometime annoing solve captcha.

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



Oct 14, 2022, 16:18 by ***@***.***:

**Comment by hapashtiepa** [comment]
I wanted tu put python 3.8 but  I have this same issue like guy in this link. GUYS thanku for app, and I will hope you wil find a way how run automated captha on newest python.  I am not gonna complicate my life with uninstaling newest python.  thanks   
https://askubuntu.com/questions/1372580/complete-uninstall-of-python3-10

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



Oct 14, 2022, 16:18 by ***@***.***:

**Comment by jan-provaznik** [comment]
The current version has been updated to work with regular tensorflow installation, no need to downgrade your python version to be able to use the tflite_runtime package.

**Comment by hapashtiepa** [comment]
Thanks again. I installed tensorflow, but I do not know how install tflite. I have tried all wheels, still the same answer {wheel is not supported on this platform| I have Kali Linux.  https://github.com/google-coral/pycoral/releases/  I Give up.  but thank you.  Your are Doing great Job and I am not profesional.

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



Oct 27, 2022, 18:22 by ***@***.***:

**Comment by jan-provaznik** [comment]
With tensorflow installed you do not need tflite. That is the whole point.
Try running the ulozto-downloader with the `--auto-captcha`.

```
ulozto-downloader --auto-captcha --parts 5 ...
```

**Comment by hapashtiepa** [comment]
printscreen in atachment,

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



Oct 28, 2022, 17:45 by ***@***.***:

**Comment by cobayashi7411** [comment]
link on printscreen. https://ww7.io/TkDULj

**Comment by jan-provaznik** [comment]
Right. When people say _try the latest version_ in issues, they often mean not the latest release, but the latest commit in the main development branch. This is what I meant. In this case you should do something along the lines of

```
pip3 install --force-reinstall git+https://github.com/setnicka/ulozto-downloader
```

which will install the latest version of the project. The `--force-reinstall` flag is important here since the current development version is the same as the latest release and the package manager would refuse to do anything. Oh, you might have to install git first (possibly through `apt install git` on Kali).

**Comment by SpiReCZ** [comment]
FYI: if someone has issue with specific newer Python version or anything regarding installation... there is a possibility to run ulozto-downloader in Docker.. That way you will use Python version of your choosing.

**Comment by cobayashi7411** [comment]
> Right. When people say _try the latest version_ in issues, they often mean not the latest release, but the latest commit in the main development branch. This is what I meant. In this case you should do something along the lines of
> 
> ```
> pip3 install --force-reinstall git+https://github.com/setnicka/ulozto-downloader
> ```
> 
> which will install the latest version of the project. The `--force-reinstall` flag is important here since the current development version is the same as the latest release and the package manager would refuse to do anything. Oh, you might have to install git first (possibly through `apt install git` on Kali).

fhm. the resul of reinstaling with this link is result here  https://ibb.co/DQkR91B  and next try to download movie and result is different but not working.  result is here  https://imgbb.com/

**Comment by setnicka** [comment]
I just released a new version 3.2.0 to the PyPI, it could be installed by just:
```shell
pip3 install --upgrade ulozto-downloader
```
It should be similar to version you downloaded directly from the repository, but with some minor fixes.

You didn't send the last screenshot, it is just a link to imgbb.com. Could you please try the just released 3.2.0 version and send a screenshot if any problem occurs?

**Comment by cobayashi7411** [comment]
> I just released a new version 3.2.0 to the PyPI, it could be installed by just:
> 
> ```shell
> pip3 install --upgrade ulozto-downloader
> ```
> 
> It should be similar to version you downloaded directly from the repository, but with some minor fixes.
> 
> You didn't send the last screenshot, it is just a link to imgbb.com. Could you please try the just released 3.2.0 version and send a screenshot if any problem occurs?

Newest version 3.2.0 with full fat tensorflow WORKS like a charm. Really like you. Now I am gona play with it in termux. btw Vzum worked in Germany, where ulozto is blocked. I had some issue with this, but I did not get it why when it works with Tor. I will be there in January let you know. :dancing_men:

**Comment by setnicka** [comment]
It looks solved now, closing this issue.
